### PR TITLE
Intergrate d3

### PR DESCRIPTION
**Pull request by fanonxr**

### CODE REVIEW

File: resources/scripts/app.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+for row in edges_df.itertuples(index=True, name="Pandas"):
+    source = getattr(row, "source")
+    target = getattr(row, "target")
+    weight = getattr(row, "size")
+    label = getattr(row, "label")
+
+    # create the edge
+    edge = graph.Edge(source=source, target=target, directed=False, weight=weight, label=label)
+
```
Comment (shreyapandit) [inline resources/scripts/app.py:52]: Do we still need this file?

File: visualization/js/f-d3.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
+let width = 1400;
+let height = 800;
+
+
+let color = d3.scaleOrdinal(d3.schemeCategory10);
```
Comment (shreyapandit) [inline visualization/js/f-d3.js:1]: I don't think this color is used anywhere. @fanonxr could you remove it, if so?

**Review by shreyapandit — CHANGES_REQUESTED** [review]
Minor feedback, otherwise looks good!

**Comment by shreyapandit** [comment]
@fanonxr  Can you also change the centering and zoom so that the initial load of the graph doesn't go outside the page, like so?

Screen Shot 2019-07-28 at 3 28 56 PM
Thanks!

File: resources/scripts/app.py (hunk 1 above)
Comment (fanonxr) [reply resources/scripts/app.py:52]: Not at all. I actually just deleted that entire folder

**Review by shreyapandit — APPROVED** [review]

**Pull request merged**